mw-webapp/src/logic/wayPage/reportsTable/WayColumns.module.scss

```diff
@@ -20,8 +20,10 @@ $listLeftMargin: 20px;
   width: 20px;
   height: 20px;
   padding: 0;
+  color: transparent;
   line-height: 20px;
   text-align: center;
+  text-shadow: 0 0 0 var(--secondaryTextColor);
```

Comment:
Do we need to set specific color if other parameters is0

Reply:
yes if we doesnt set cpecific color then emoji doesnt have color

Reply:
Good